deploy/utils/tests/test_convert_api_version.py

```
@@ -0,0 +1,447 @@
+
+REPO_ROOT = _Path(__file__).resolve().parents[3]
+
+CONFORMANCE_NAMES = ["agg", "disagg", "disagg_router", "disagg_planner"]
+
+
+def _cluster_available() -> bool:
+    if shutil.which("kubectl") is None:
+        return False
+    try:
+        _sp.run(
+            ["kubectl", "get", "crd", "dynamographdeployments.nvidia.com"],
```

Ruff (0.15.15)

[error] Starting a process with a partial executable path

(S607)